src/lib/project/twitter.ex

```diff
@@ -0,0 +1,280 @@
+defmodule Project.Twitter do
+  @moduledoc """
+  The Twitter context.
+  """
+
+  import Ecto.Query, warn: false
+  alias Project.Repo
+
+  alias Project.Twitter.Post
+  alias Project.Twitter.User
+
+  @spec list_posts :: [%Post{}]
+  @doc """
+  Returns the list of posts.
+
+  ## Examples
+
+      iex> list_posts()
+      [%Post{}, ...]
+
+  """
+  def list_posts do
+    Post
+    |> Repo.all()
+    |> Repo.preload(:user)
+  end
```

Comment:
```suggestion
  def list_posts do
    Post
    |> preload(:user)
    |> Repo.all()
  end
```
Repo.preloadは使わずに、Ecto.Queryのpreloadを使いましょう！

Reply:
他のところも同様に修正しましょう